src/flownet/ahm/_ahm_iteration_analytics.py

```diff
@@ -289,10 +289,14 @@ def make_dataframe_simulation_data(
         nb_real: number of realizations
 
     """
-    iteration = sorted(
-        [int(rel_iter.replace("/", "").split("-")[-1]) for rel_iter in glob.glob(path)]
-    )[-1]
-    runpath_list = glob.glob(path[::-1].replace("*", str(iteration)[::-1], 1)[::-1])
+    if "pred" in mode:
+        runpath_list = glob.glob(path)
+        iteration = "latest"
+    else:
+        iteration = sorted(
+            [rel_iter.replace("/", "").split("-")[-1] for rel_iter in glob.glob(path)]
+        )[-1]
+        runpath_list = glob.glob(path[::-1].replace("*", str(iteration)[::-1], 1)[::-1])
```

Comment:
Check that this code works and not suffers from the same bug we had before: see for ex. https://github.com/equinor/flownet/issues/360

Reply:
Will pick this up in pull request for issue #360